k8s/client.go

```diff
@@ -448,3 +456,206 @@ func (client *Client) FindDeploymentByName(ctx context.Context, name string) (*a
 func (client *Client) FindDeploymentByLabel(ctx context.Context, label string) (*appsv1.DeploymentList, error) {
 	return client.AppsV1().Deployments(client.Namespace).List(ctx, metav1.ListOptions{LabelSelector: label})
 }
+
+func (client *Client) DeployCoreOperatorSync(ctx context.Context, coreInstance cloud.CreatedCoreInstance, version string) (*appsv1.Deployment, error) {
+	labels := client.LabelsFunc()
+	const toCloudImage = "ghcr.io/calyptia/core-operator/sync-to-cloud"
+	const fromCloudImage = "ghcr.io/calyptia/core-operator/sync-from-cloud"
+	args := []string{"-cloud_url", client.CloudBaseURL, "-token", client.ProjectToken, "-core_instance", coreInstance.Name, "-namespace", client.Namespace}
```

Comment:
We inject environment variables for all those options. Check at https://github.com/calyptia/core-operator/blob/main/cmd/sync/sync.yaml#L22

Also, include the cloud url. Lets don't use command line arguments.